CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# [MongooseIM 4.2.0](https://github.com/esl/MongooseIM/releases/tag/4.2.0) - 2021-04-20
+
+## Highlights
+
+- Prepared queries introduced
+- Inbox extensions
+- Miscellaneous enhancements & improvements
+
+## All changes
+
+### Added
+- Documentation for `mod_offline.store_groupchat_messages` option (#2992)
+- Inbox extensions (#3067)
+- `mod_domain_isolation` module to limit message passing between domains (#3070)
+
+### Removed
+- Usage of deprecated `http_uri` in `ejabberd_auth_http` and `mod_muc_room` modules (#3026)
+- Config reload functionality with the flat config format (#3030)
+
+### Changed
+- Usage of maps instead of lists in session management (#3018)
+- Improved timestamp logic (#3031)
+- Prepared queries for MS SQL, mySQL and pgSQL (#3039, #3050, #3055, #3059, #3060, #3066, #3074, #3078)
+- Improved pipeline for `mod_smart_markers` (#3068)
+- Unified checks for chat markers (#3080)
+- Set `mod_mam_meta.pm.archive_groupchats` to `false` by default (#3082)
```

Comment:
Hm... maybe this should be "fixed"? According to the docs they should've been set to 'false' already.